core/templates/dev/head/css/oppia.css

```diff
@@ -4438,7 +4438,7 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-donation-card-content-wide {
-  margin: 0 auto;
+  margin: 40px auto;
```

Comment:
@anthkris `margin: 40px auto 80px;` Updated the commit. Missed spectating bottom margin in desktop view. Thanks